core/whistle_number_manager-1.0.0/src/carriers/wnm_bandwidth2.erl

```diff
@@ -44,6 +44,26 @@
 -define(BW_ORDER_NAME_PREFIX,
         whapps_config:get_binary(?WNM_BW_CONFIG_CAT, <<"order_name_prefix">>, <<"Kazoo">>)).
 
+-define(IS_US_TOLLFREE(Prefix),
+    Prefix == <<"800">> orelse
+    Prefix == <<"822">> orelse
+    Prefix == <<"833">> orelse
+    Prefix == <<"844">> orelse
+    Prefix == <<"855">> orelse
+    Prefix == <<"866">> orelse
+    Prefix == <<"877">> orelse
+    Prefix == <<"880">> orelse
+    Prefix == <<"881">> orelse
+    Prefix == <<"882">> orelse
+    Prefix == <<"883">> orelse
+    Prefix == <<"884">> orelse
+    Prefix == <<"885">> orelse
+    Prefix == <<"886">> orelse
+    Prefix == <<"887">> orelse
+    Prefix == <<"888">> orelse
+    Prefix == <<"889">>
+).
```

Comment:
Yes this is for 3.22 as the defect was reported against it.  I will open a second PR for master shortly.